UtilityMemoryBag.lua

```diff
@@ -1,7 +1,127 @@
 -- Utility memory bag by Directsun
--- Version 2.6.0
+-- Version 2.7.0
 -- Fork of Memory Bag 2.0 by MrStump
 
+CONFIG = {
+    MEMORY_GROUP = {
+        -- Use an identical value across bags in the group.
+        -- Set this to 'nil' in order to not have the bag in a group.
+        NAME = nil,
+
+        -- This determines how many frames to wait before actually placing objects onto the table when the "Place" button is clicked.
+        -- This gives the other bags time to recall their objects.
+        -- The delay ONLY occurs if other bags have objects out.
+        FRAME_DELAY_BEFORE_PLACING_OBJECTS = 30,
+    },
+}
+
+
+--[[ Memory Bag Groups ]]-------------------------------------------------------
+
+-- Click the "Recall" button on all other bags in my memory group.
+function recallOtherBagsInMyGroup()
+    for bagGuid,_ in pairs(GlobalMemoryGroups:getGroup(CONFIG.MEMORY_GROUP.NAME)) do
+        if bagGuid ~= self.getGUID() then
+            bag = getObjectFromGUID(bagGuid)
+            bag.call('buttonClick_recall')
+        end
+    end
+end
+
+-- Return "true" if another bag in my memory group has any objects out on the table.
+function anyOtherBagsInMyGroupArePlaced()
+    for bagGuid,_ in pairs(GlobalMemoryGroups:getGroup(CONFIG.MEMORY_GROUP.NAME)) do

```

Comment:
if using nongroup, would have to check for special case somewhere. Maybe a flag.